pkg/cmd/cli/describe/deployments.go

```diff
@@ -96,25 +108,28 @@ func (d *DeploymentConfigDescriber) Describe(namespace, name string, settings kc
 				formatString(out, "Latest Deployment", fmt.Sprintf("error: %v", err))
 			}
 		} else {
-			header := fmt.Sprintf("Deployment #%d (latest)", deployutil.DeploymentVersionFor(deployment))
-			printDeploymentRc(deployment, d.kubeClient, out, header, true)
+			header := fmt.Sprintf("Deployment #%d", deployutil.DeploymentVersionFor(deployment))
+			if deployment.Name == activeDeploymentName {
+				header += " (latest)"
```

Comment:
Active != latest - you should print latest only for the latest deployment -failed or complete